codegen/src/generator/json.rs

```diff
@@ -102,7 +102,11 @@ fn generate_method_signature(operation: &Operation) -> String {
 
 fn generate_payload(operation: &Operation) -> String {
     if operation.input.is_some() {
-        "let encoded = serde_json::to_string(input).unwrap();
+        "let mut encoded = serde_json::to_string(input).unwrap();
+         if encoded == \"null\" {
+            encoded = \"{}\".to_owned();
+         }
+         debug!(\"Setting payload to '{}'\", encoded);
```

Comment:
I see that we have some unit-struct requests being generated, so this is probably still relevant. However, I think a better fix would be to just not generate unit structs for requests that have no members. I believe this is already being done for at least one of the generated protocols.